### ISSUE TITLE
Julie's upcoming updates

### ISSUE THREAD

**Issue by jules32** [Openscapes/website-new#18]
Hi @samanthacsik and @stefaniebutland, @ileanaf and I are learning about user centered design and have had a few test users at our site! I will go through on Monday and write up more details and edit some of the content accordingly (reduce text generally overall!) in a branch and PR. Some guiding thoughts for us and starting TODO's for me: 

Goals for users to encounter:

1. Finding the resources with a small amount of information (researchers > Champions curric, 1 min)
2. Someone else learning about this and seeing information that shows the value (managers > testimonials, 1 min)
3. Stories - emphasize (20 min)

Starting ideas to change
- [x] testimonials on p 1, not events
- [x]  initiatives on single page like current live version (comparative, more pleasant for users)
- [x]  language: "curriculum" vs "open educational resources" (no one knows this word)
- [x]  events page - downplay, "our events" (since this isn't a primary way to engage - curriculum and media are)
- [x]  "stories" - vs blog page
- [x]  impact page - stories, logos, quotes, highlight 
- [x]  connect update (sponsor language, feature comparison table someday)

- [x] Navbar (v means collapsed menu) - Search bar changes the way people use the site 
HOME • ABOUT v • INITIATIVES • IMPACT • BLOG • GET INVOLVED v

**Comment by samanthacsik** [comment]
Super exciting @jules32! I'll be on the lookout for the PR to explore all the updates :)

**Comment by jules32** [comment]
Hi! I've made a PR – it still has some work in progress for me to do but also places that @samanthacsik can lead (I've created separate issues for those). 

When you have a look, will you share any reactions/ideas you have. And see if you have any ideas for how to stylize quotes? Thinking in particular: 

- impacts page
- index page "growing the open movement"

I'll be able to work more on this on Friday (if not before). Thank you so much!
